src/gui/qgsfeaturelistcombobox.cpp

```diff
@@ -53,6 +53,7 @@ QgsFeatureListComboBox::QgsFeatureListComboBox( QWidget *parent )
 
   mLineEdit = new QgsFilterLineEdit();
   mLineEdit->setSelectOnFocus( true );
+  connect( mLineEdit, &QgsFilterLineEdit::cleared, this, &QgsFeatureListComboBox::setValueToNull );
```

Comment:
I tend to prefer a lambda in such a simple case, when the slot is only a few lines and it is just used once as a slot, but it's really up to you.

Reply:
I agree. When I made it like this, I wasn't sure yet that it's only used once.